README.md

```diff
@@ -225,8 +225,10 @@ The following inputs are for controlling the behavior of this GitHub Actions,
 regardless of the authentication mechanism.
 
 -   `project_id`: (Optional) Custom project ID to use for authentication and
-    exporting into other steps. If unspecified, the project ID will be extracted
-    from the Workload Identity Provider or the Service Account Key JSON.
+    exporting into other steps. If unspecified, we will attempt to extract the
+    project ID from the Workload Identity Provider, Service Account email, or
```

Comment:
Is this actually accurate though? It doesn't look like it ever looks in the "Workload Identity Provider".

Reply:
In the future, you will be able to use project IDs in the WIF ID; that's not available today.

Reply:
You're the expert here — I only learned that any of this stuff existed yesterday :) (But as a user I would probably be more interested in docs that teach me what to do today than that describe future possibilities non-explicitly.)